tool/run/run

```diff
@@ -940,6 +940,29 @@ proc build_core_image { modules } {
 	exec cp -f [run_dir]/genode/config [run_dir].config
 }
 
+proc build_initrd { modules } {
+    
+    copy_genode_binaries_to_run_dir $modules
+    
+    set modules [glob -nocomplain -tails -directory [run_dir]/genode/ *]
+    set excluded_modules [kernel_files]
+	
+    foreach file [glob -nocomplain [run_dir]/genode/*.config] {
+		check_xml_syntax $file }
+    
+    exec cp -f [run_dir]/genode/config [run_dir].config
+
+    set here [pwd]
+    cd [run_dir]
+    puts "generating initrd"
+    exec cp genode/initramfs init
+    exec mkdir tmp
+    set files "init\ntmp\ngenode\n"
+    append files [exec find genode -type f,l -printf "genode/%f\n"]
+    exec -ignorestderr echo $files | cpio -o -L -H newc > initrd
```

Comment:
Here, it would be good to wrap `cpio` into the `[installed_command]` function. See https://github.com/genodelabs/genode/commit/10485b0d39f2dc5d2a8a555449007055e7029324